homeassistant/components/plex/__init__.py

```diff
@@ -56,40 +43,6 @@
 from .errors import ShouldUpdateConfigEntry
 from .server import PlexServer
 
-MEDIA_PLAYER_SCHEMA = vol.All(
-    cv.deprecated(CONF_SHOW_ALL_CONTROLS, invalidation_version="0.110"),
```

Comment:
If you can I think it is good to keep the deprecated part for a few releases if people are late to upgrade

Reply:
If they're on 0.100 or newer then the YAML config be removed with no (🤞) impact. That covers anyone that's on a version released in the last ~8 months.

Reply:
Let's remove it, since it is already been raised as invalid. Nothing won't change there.